### DISCUSSION TITLE
Meet the LtC Public Review Team

### DISCUSSION THREAD

**Discussion by rondlg** [tdwg/ltc#56] [Announcements]
In case you were wondering who we are. The motley crew that has been fishing the waters of collections description standards is:

- Sarah Vincent
- Matt Woodburn
- Janeen Jones
- Kate Webbink
- Jutta Buschbom
- Sharon Grant
- Maarten Trekels 

With the help of Ben Norton, and we'll be working through the Public Review with you all.

**Comment by jbstatgen** [comment]
We are ratified!
